src/frontend/components/TranscriptSentence/index.tsx

```diff
@@ -30,19 +30,13 @@ export const TranscriptSentence = ({
 
   const textSentence = () => ({ __html: `${cue.text} ` });
 
-  if (active) {
-    return (
-      <ActiveSentence
-        onClick={() => setTime(cue.startTime)}
-        dangerouslySetInnerHTML={textSentence()}
-      />
-    );
-  } else {
-    return (
-      <Sentence
-        onClick={() => setTime(cue.startTime)}
-        dangerouslySetInnerHTML={textSentence()}
-      />
-    );
-  }
+  const SentenceComponent = active ? ActiveSentence : Sentence;
+
+  return (
+    <SentenceComponent
+      className={active ? 'active-sentence' : ''}
```

Comment:
thanks for your PR @flo-pereira as this is a much expected feature!

I would like to bikeshed a little on the name of this class: should it be "sentence-active" to go from the most generic to the most specific? Should the non active sentence get nothing or should it get a "sentence-xxxxx" class?

Reply:
fixed 5e0d27734e2492f1522f048f14429e169082b0c4.

Hi @sampaccoud, non active sentence doesn't require specific class. class is just used a marker that let element being target.

Reply:
:+1: